app.py

```diff
@@ -487,12 +492,70 @@ def print_events(topic=Publisher.AUTO_TOPIC, **msg_data):
     """
     utils.debug("%s\n\tParameters: %s" % (topic, msg_data))
 
+def setup_remote_host(remote_host):
```

Comment:
I think this part can be moved to new file inside of `invesalius/net`. I think the file should be named something like `remote_control.py`. Also, I think it's better to use a class.

Reply:
Thanks for the suggestion! Done.